### ISSUE TITLE
SMA Sunnyportal Homemanager

### ISSUE THREAD

**Issue by fritserasmus** [rlindner/node-red-contrib-sma-webconnect#3]
Roland,
Apologies for posting this here, but I was not sure if you were still working on that project.

I have a rather new setup with Sunny Island and Home Manager 2.0 and was hoping it will be possible to get some info via Home Manager like production, utilization, battery status: especially battery status.

Kind Regards

Frits

**Comment by rlindner** [comment]
Hey Frits,

as you've written [here](https://gist.github.com/ImUrlaub/90b104ba6b1aee4e090fd34d651f0e69#gistcomment-3255612), you have some kind of "SunnyBoys". I've just released a [new version 0.1.0](https://github.com/rlindner/node-red-contrib-sma-webconnect/releases/tag/v0.1.0) of the node which adds support for the SUNNY BOY STORAGE 2.5. So maybe this could be helpful for you.

If you have any question please just reopen the issue.

Roland

**Comment by fritserasmus** [comment]
@rlindner ,
I gave up on that for a while since I had some issues with my SMA Sunny Island and then it was work that got the better of me.
I have been spending some time to do some catch up on previous chats and where I should jump in and try to continue.

I still have the two  Sunny Boy 4000TL-21 and the Sunny Island 8.0H just working fine.
Load shedding in South Africa becomes a real pain and I have to diligently monitor batteries/PV production/use.

I imported the code on this url; and updated my UID and PW  
[https://flows.nodered.org/flow/90b104ba6b1aee4e090fd34d651f0e69/in/4uzrof3bLPmN](url)

and then got this error
![image](https://user-images.githubusercontent.com/7281074/220593772-0b2ec23e-5cd4-4d61-9575-5c1d6cf111c2.png)

![image](https://user-images.githubusercontent.com/7281074/220593459-5872fd04-7c26-41df-89ac-4a477897b751.png)

![image](https://user-images.githubusercontent.com/7281074/220593361-a1ef48b9-630a-4dbe-a803-30a56e169986.png)


 
Am I doing something wrong here?

Frits
